tracker/src/tracker-extract/90-libav-audio-generic.rule

```diff
@@ -1,4 +1,4 @@
 [ExtractorRule]
 ModulePath=libextract-libav.so
-MimeTypes=audio/*;
+MimeTypes=audio/*;application/ogg;
```

Comment:
This looks dangerous since we may try to extract and tag any application/ogg file as an audio file, even when they can contain other media streams.

If it is not supported by our shared-mime-info package, I would recommend to give a look to:
https://launchpadlibrarian.net/167465303/0001-Add-audio-x-opus-ogg-mime-type.patch